src/wallet/init.cpp

```diff
@@ -227,8 +228,11 @@ bool WalletInit::Open() const
         return true;
     }
 
+    uint64_t wallet_creation_flags =
```

Comment:
I'd prefer to remove this, and make the flag settable through the `createwallet` and `setwalletflag` options (see how `WALLET_FLAG_DISABLE_PRIVATE_KEYS` is set)

Reply:
Above change resulted in its removal.